src/download.js

```diff
@@ -24,4 +30,4 @@ function DownloadFile(path, skylink, opts) {
   });
 }
 
-module.exports = { DownloadFile };
+module.exports = { DefaultDownloadOptions, DownloadFile };
```

Comment:
I don't think we need to export `DefaultDownloadOptions` - they will be merged in if `customOptions` are specified.

Reply:
These variables should follow camel case convention then.